### PR TITLE
Fix REXX/solution_1 Docker lint warning

### PR DESCRIPTION
**Pull request by rbergen**
Fixes apt => apt-get lint warning in PrimeREXX/solution_1/Dockerfile

### CODE REVIEW

File: PrimeREXX/solution_1/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM ubuntu:20.04
-RUN apt update && apt upgrade -y \
-	&& apt install -y regina-rexx
+RUN apt-get update && apt-get upgrade -y \
```
Comment (marghidanu) [inline PrimeREXX/solution_1/Dockerfile]: Why is the `apt-get upgrade` required here?

Comment (rbergen) [reply PrimeREXX/solution_1/Dockerfile]: Good question, to which I don't have the answer. I focused on the lint warning, which required apt to be replaced by apt-get. I think we will need input from @jossRRM to answer your question.

Comment (rbergen) [reply PrimeREXX/solution_1/Dockerfile]: I decided to just remove the apt upgrade and see what happened. The solution still builds and runs, so I've committed that change.

**Review by jossRRM — COMMENTED** [review]
I was under the impression that apt install and apt upgrade are the new (and accepted) commands. Was I wrong? reference: https://phoenixnap.com/kb/apt-vs-apt-get

I have an upcoming pull request for some changes (cosmetic) to the code and one for a solution_2 that uses NetRexx, a compiled language that is based on Rexx, with classes and arrays.  Should I make your changes to my Dockerfile? Also, is the upgrade not needed?

EDIT:
I just saw your emails regarding this. Since replacing the apt with apt-get and removing apt upgrade work, I see no difficulties with these changes. I will make them to my local and repository Dockerfile(s).

**Comment by rbergen** [comment]
> I was under the impression that apt install and apt upgrade are the new (and accepted) commands. Was I wrong?

I don't think it's a black-and-white right vs wrong thing. I believe consensus (still) is that apt-get is the command to use in scripted scenario's, which a Dockerfile obviously really is. In any case, with the consent you've now given I will indeed merge this.

**Comment by jossRRM** [comment]
Hi, Rutger. I tried to create a pull request from my repository. I had changes to solution_1 and a new solution_2, both under the PrimeREXX directory. When I tried to point to my repository both solutions were added. But the solution_2 uses a different REXX dialect and I think it should be in a
separate pull request. Is there a way to separate the two solutions into two separate requests without making a new directory under Primes for solution_2?
Joe Ossino (jossRRM)
---------------------------- Original Message ----------------------------

Subject: Re: [PlummersSoftwareLLC/Primes] Fix REXX/solution_1 Docker lint warning (#479)

From: "Rutger van Bergen" ***@***.***>

Date: Sun, July 25, 2021 6:08 pm

To: "PlummersSoftwareLLC/Primes" ***@***.***>

Cc: "jossRRM" ***@***.***>

"Mention" ***@***.***>

--------------------------------------------------------------------------

**Pull request merged**